aerolyzer/horizon.py

```diff
@@ -1,45 +1,35 @@
-import sys
 import cv2
-import os
+import sys
 import numpy as np
-from matplotlib import pyplot as plt
 
-def test():
-	print "hi"
-def _testAnalyze(self, directory):
-	included_extensions = ['jpg','JPG']
-	files = [fn for fn in os.listdir(directory) if any(fn.endswith(ext) for ext in included_extensions)]
-	for image in files:
-		img_noblur = cv2.imread(str(directory)+image)
-		img_noblur = cv2.cvtColor(img_noblur, cv2.COLOR_BGR2GRAY)
-		self.getHorizon(str(directory), img_noblur,image)
+	
+def Horizon(directory, image, output=0):
+	imgOg = cv2.imread(str(directory)+image) # Read image
+	
+	img_YUV = cv2.cvtColor(imgOg,cv2.COLOR_BGR2YCR_CB)	# Convert from BGR to YCRCB
+	b, r, g = cv2.split(img_YUV) 						# Split into blue-green-red channels
+	imgBlueEqu = cv2.merge((cv2.equalizeHist(b), r, g))	# Equalize Blue Channel & Merge channels back
```

Comment:
This variable (imgBlueEqu) is never used.

Reply:
Why equalize on the Y (brightness) channel?

Reply:
I made a syntax mistake on line 11. 

The overall purpose of these are to normalize brightness variance between different lighting conditions in different images. Further research shows that my previous approach was unnecessary. I have now simplified lines 9-12 to use opencv's histogram equalization function.  

https://github.com/lamki866/Aerolyzer/blob/b154d392e6d99c506cbc7bb002d14fca7412585c/aerolyzer/horizon.py#L21